tests/scripts/thread-cert/node.py

```diff
@@ -835,6 +835,34 @@ def srp_client_stop(self):
         self.send_command(f'srp client stop')
         self._expect_done()
 
+    def srp_client_get_state(self):
+        cmd = 'srp client state'
+        self.send_command(cmd)
+        return self._expect_command_output(cmd)[0]
+
+    def srp_client_get_auto_start_mode(self):
+        cmd = 'srp client autostart'
+        self.send_command(cmd)
+        return self._expect_command_output(cmd)[0]
```

Comment:
```suggestion
        return self._expect_command_output(cmd)[0] == 'Enabled'
```

nit: better return True or False

Reply:
I noticed the common pattern in `node.py` seems to be to use `get_{something}` over `is_{something}_enabled` and return the value from the corresponding CLI command. 

So to be consistent I would prefer to continue to use `get_auto_start_mode()` over `is_auto_start_mode_enabled()` and using `get_mode()` I think better to keep it string over bool return type?